keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/Capacity.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.clouddriver.model
+
+data class Capacity(
+  val min: Int,
+  val max: Int,
+  val desired: Int?
```

Comment:
```suggestion
  val desired: Int? = min
```

Reply:
Is there any point doing this in the Clouddriver API? Is desired sometimes not returned by Clouddriver?

Reply:
Dunno... 🤷‍♂️